lib/fauxpaas/artifact_builder.rb

```diff
@@ -38,5 +39,12 @@ def add_reference(ref, path)
         .write
     end
 
+    def install_local_gems(path)
+      raise "bundler failed to install gems in #{path}" unless Dir.chdir(path) do
+        _, _, status = Fauxpaas::system_runner.run("bundle install --deployment")
```

Comment:
I think it's fine to use the service locator, but I'd rather see it resolved once, at initialization. That is, `attr_reader :system_runner` with `@system_runner = Fauxpaas.system_runner`. Tests can register/unregister a special runner if necessary. If we find that we're doing a bunch of before/afters, we could add it as an injectable collaborator.

Separately, sort of a taste matter: I think we should use `.` over `::` to signify that this is a message send and that `Fauxpaas` is functioning more like a class than a module here.

Reply:
Oh, I see that Code Climate also flagged the `::` method call -- let's follow that and use `.`.